### ISSUE TITLE
Cloning a destination does not clone the selected resource IDs

### ISSUE THREAD

**Issue by rbeckman-nextgen** [nextgenhealthcare/connect#3594]
This is easily reproducible by creating a new channel, then cloning a destination. The default resource (or whatever is selected) isn't copied to the new destination.

Imported Issue. Original Details:
Jira Issue Key: MIRTH-3707
Reporter: narupley
Created: 2015-04-28T13:44:58.000-0700

**Comment by rbeckman-nextgen** [comment]
Fixed in revisions 7675/7676. After cloning a destination, the resource IDs will now be correctly added to the cached map.

Imported Comment. Original Details:
Author: narupley
Created: 2015-04-28T13:52:06.000-0700

**Comment by rbeckman-nextgen** [comment]
To replicate this issue, I created a simple channel and cloned the destination. I verified within the Libraries on the summary tab that the default resources were not selected. I also inspected the XML to verify that it wasn't contained here as well. I then repeated this steps in a new build and was able to see that the resource IDs did get selected this time and were contained within the channel XML.

Imported Comment. Original Details:
Author: jaysenp
Created: 2015-05-20T15:51:33.000-0700
